src/Controller/Admin/WorkflowController.php

```diff
@@ -166,8 +166,10 @@ public function submitGlobalAction(Request $request, Registry $workflowRegistry,
         $workflowOptions = $request->get('workflow', []);
         $workflow = $workflowRegistry->get($this->element, $request->get('workflowName'));
 
+        $saveSubject = $workflowManager->getGlobalAction( 'product_status_workflow', $request->get('transition') )->getSaveSubject() ?? true;
```

Comment:
hardcoded name of a workflow? that can't be right here.

Reply:
Sorry, that slipped by when I was pulling this from my project into this PR, I will change it and update the PR